tests/RobotFramework/tests/cumulocity/configuration/configuration_operation.robot

```diff
@@ -108,6 +108,30 @@ Manual config_snapshot operation request
     ...    expected_status=successful
     ...    c8y_fragment=c8y_UploadConfigFile
 
+Trigger config_snapshot operation from another operation
+    Set Device Context    ${PARENT_SN}
+    Publish and Verify Local Command
+    ...    topic=te/device/main///cmd/sub_config_snapshot/local-1111
+    ...    payload={"status":"init","tedgeUrl":"http://${PARENT_IP}:8000/tedge/file-transfer/${PARENT_SN}/sub_config_snapshot/local-1111","type":"tedge-configuration-plugin"}
+    ...    expected_status=successful
+    ...    c8y_fragment=c8y_UploadConfigFile
+    ${snapshot}    Execute Command    curl http://${PARENT_IP}:8000/tedge/file-transfer/${PARENT_SN}/sub_config_snapshot/local-1111
+    ${config}      Get File    ${CURDIR}/tedge-configuration-plugin.toml
+    Should Be Equal    ${snapshot}     ${config}
+
+Trigger custom config_snapshot operation
+    [Teardown]    Restore config operations
+    Customize config operations
+    Set Device Context    ${PARENT_SN}
+    Publish and Verify Local Command
+    ...    topic=te/device/main///cmd/sub_config_snapshot/local-1111
```

Comment:
```suggestion
    ...    topic=te/device/main///cmd/config_snapshot/local-1111
```
Right?

Reply:
You are correct

Reply:
Fixed by https://github.com/thin-edge/thin-edge.io/pull/2844/commits/2457d5bec9a6eb0441ebc97f9279fa010ea2b06b